server/src/test/java/io/crate/integrationtests/ExplainAnalyzeIntegrationTest.java

```diff
@@ -88,4 +90,80 @@ public void testExplainSelectWithoutJobExecutionContexts() {
             .containsKeys("Total")
             .satisfies(m -> assertThat(m.keySet()).containsAnyOf(nodeIds.toArray(new String[] {})));
     }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void test_explain_analyze_on_statement_with_table_function_subquery() {
```

Comment:
Not sure if this test case adds any value (it was just the first one I run while developing). Maybe remove?

Reply:
Hm, `select *, (select 1) from locations` will actually result in similar output. Why would you see this more valuable?

Reply:
It was just a question, but yeah, doesn't go into some other code path, I'd say to just remove this one.